production-versions.ini

```diff
@@ -11,3 +11,4 @@ liquid-dokuwiki = liquidinvestigations/liquid-dokuwiki:0.1.0
 liquid-nextcloud = liquidinvestigations/liquid-nextcloud:0.2.3
 rocketchat = rocketchat/rocket.chat:3.9.1
 rocketchat-mongo = mongo:4.4
+thumbnail-service = fpurchess/preview-service:latest 
```

Comment:
Let's link some specific version instead of `latest`. Using the `latest` tag means any breaking change they ever add will break our apps too. That's what tags are for.

If they don't tag their containers, you can use a `sha3`  docker image link instead.